lib/monitoring/aws-appsync/AppSyncMetricFactory.ts

```diff
@@ -9,6 +8,26 @@ import {
 
 const Namespace = "AWS/AppSync";
 
+/**
+ * Partial copy of [GraphqlApi] from "@aws-cdk/aws-appsync-alpha"
+ *
+ * @experimental Subject to change to match the alpha module.
+ */
+export interface GraphqlApi {
```

Comment:
How does this work with JSII? The interfacing would not be similar to TypeScript where it infers the members.

Reply:
Good point, it probably doesn't work in languages like Java.